packages/components/package.json

```diff
@@ -72,27 +72,27 @@
     "@glint/environment-ember-loose": "^1.4.0",
     "@glint/template": "^1.4.0",
     "@rollup/plugin-babel": "^6.0.4",
-    "@tsconfig/ember": "^3.0.6",
+    "@tsconfig/ember": "^3.0.8",
     "@types/ember-qunit": "^6.1.1",
     "@types/ember-resolver": "^9.0.0",
     "@types/ember__destroyable": "^4.0.5",
-    "@types/qunit": "^2.19.7",
-    "@types/rsvp": "^4.0.6",
-    "@typescript-eslint/eslint-plugin": "^6.14.0",
-    "@typescript-eslint/parser": "^6.14.0",
+    "@types/qunit": "^2.19.10",
+    "@types/rsvp": "^4.0.9",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
+    "@typescript-eslint/parser": "^6.21.0",
     "babel-plugin-ember-template-compilation": "^2.2.4",
     "concurrently": "^8.2.2",
     "ember-basic-dropdown": "^8.1.0",
     "ember-concurrency": "^4.0.2",
     "ember-template-lint": "^4.18.2",
     "ember-template-lint-plugin-prettier": "^4.2.0",
-    "eslint": "^8.52.0",
-    "eslint-config-prettier": "^8.10.0",
-    "eslint-plugin-ember": "^11.11.1",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^11.12.0",
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-n": "^17.7.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-qunit": "^8.0.1",
+    "eslint-plugin-qunit": "^8.1.1",
     "npm-run-all": "^4.1.5",
```

Comment:
I believe we can now remove `npm-run-all` as we rely on `concurrently` to run commands